custom_components/ble_monitor/ble_parser/__init__.py

```diff
@@ -146,6 +147,10 @@ def parse_data(self, data):
                     if comp_id == 0x4C and man_spec_data[4] == 0x02:  # iBeacon
                         sensor_data, tracker_data = parse_ibeacon(self, man_spec_data, mac, rssi)
                         break
+                    elif man_spec_data[0] == 0x1B and man_spec_data[1] == 0xFF \
```

Comment:
I don't think you need `man_spec_data[1] == 0xFF` in this line, as `0xFF` in this position is by definition a packet with `manufacturer specific data`

Reply:
yes, the tests pass, there should be no problems.

Reply:
https://github.com/custom-components/ble_monitor/pull/655